mad-dog/src/index.js

```diff
@@ -59,7 +59,7 @@ const makeTest = (name, testFn, { numUsers, numCreatorNodes }) => {
 }
 
 const testRunner = async tests => {
-  let failedTests = []
+  const failedTests = []
```

Comment:
let is actually correct here. since this will be modified it should not be const